packages/components/psammead-story-promo-list/README.md

```diff
@@ -0,0 +1,75 @@
+# psammead-story-promo-list - [![Known Vulnerabilities](https://snyk.io/test/github/bbc/psammead/badge.svg?targetFile=packages%2Fcomponents%2Fpsammead-story-promo-list%2Fpackage.json)](https://snyk.io/test/github/bbc/psammead?targetFile=packages%2Fcomponents%2Fpsammead-story-promo-list%2Fpackage.json) [![Storybook](https://raw.githubusercontent.com/storybooks/story-promo-list/master/badge/badge-storybook.svg?sanitize=true)](https://bbc.github.io/psammead/?path=/story/story-promo-list--default) [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/bbc/psammead/blob/latest/LICENSE) [![npm version](https://img.shields.io/npm/v/@bbc/psammead-story-promo-list.svg)](https://www.npmjs.com/package/@bbc/psammead-story-promo-list) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/bbc/psammead/blob/latest/CONTRIBUTING.md)
+
+## Description
+
+The `@bbc/psammead-story-promo-list` package is a set of two components, `StoryPromoUl` and `StoryPromoLi`. They use a `ul` and `li` HTML element respectively.
+
+## Installation
+
+`npm install @bbc/psammead-story-promo-list`
+
+## Props
+
+| Argument | Type | Required | Default | Example        |
+| -------- | ---- | -------- | ------- | -------------- |
+| children | node | Yes      | N/A     | `<StoryPromoLi><StoryPromo image={Image} info={Info} /></StoryPromoLi>` |
+
+## Usage
+
+Commonly used alongside [`psammead-story-promo`](https://github.com/BBC-News/psammead/tree/latest/packages/components/psammead-story-promo).
+
+```jsx
+import React, { Fragment } from 'react';
+import StoryPromo, { Headline, Summary } from '@bbc/psammead-story-promo';
+import { StoryPromoLi, StoryPromoUl } from '@bbc/psammead-story-promo-list';
+import { latin } from '@bbc/gel-foundations/scripts';
+
+const Image = (
+  <img src="https://foobar.com/image.jpg" />
+);
+
+const Info = (
+  <Fragment>
+    <Headline script={latin}>The headline of the promo</Headline>
+    <Summary script={latin}>The summary of the promo</Summary>
+    <time>12 March 2019</time>
+  </Fragment>
+);
+
+<StoryPromo
+  image={Image}
+  info={Info}
+/>
+
+<StoryPromoUl>
+  <StoryPromoLi>
+    <StoryPromo image={Image} info={Info} />;
+  </StoryPromoLi>
+  <StoryPromoLi>
+    <StoryPromo image={Image} info={Info} />;
+  </StoryPromoLi>
+</StoryPromoUl>
+
+```
+
+### When to use this component
+
+This component is designed to be used with story promos on 'index' pages.
+
+<!-- ### When not to use this component -->
+
+<!-- ### Accessibility notes -->
```

Comment:
You've added roles to the list and listitem elements, which is great. This should be explained here in the Accessibility notes, so that users of this package understand the reasoning for this. 

Since this is a new component, it should go through an accessibility swarm. This is to check this assistive technologies to ensure that it is working as expected, prior to merging. For example, that the list and list  items are being read out correctly by screenreaders. 
This swarm should be done prior to merge, since once merged, it will be published to npm and be available for others to use.


Here are instructions for how to use the various assistive technologies that we support: https://bbc-news.github.io/accessibility-news-and-you/accessibility-and-supported-assistive-technology

If you have any questions, please let me know.